.github/workflows/pr-frontend-unit-tests.yml

```diff
@@ -90,6 +90,23 @@ jobs:
         TEST_SHARD: ${{ matrix.chunk }}
         TEST_SHARD_TOTAL: 8
 
+  frontend-decoupled-plugin-tests:
+    needs: detect-changes
+    if: github.event_name == 'pull_request' && needs.detect-changes.outputs.changed == 'true'
```

Comment:
you don't want these to run in `main` too?

Reply:
Should be fixed in [this commit](https://github.com/grafana/grafana/pull/107942/commits/60c4b43baf4ec8f7b84b8f9b6bc16a4967949782) @kminehart